src/probnum/filtsmooth/filtsmoothposterior.py

```diff
@@ -32,3 +32,16 @@ def __len__(self):
     def __getitem__(self, idx):
         """Return the corresponding index/slice of the discrete-time solution"""
         raise NotImplementedError
+
+    def sample(self, locations=None, size=None):
+        """
+        Draw samples from the filtering/smoothing posterior.
+
+
+        If location is None, it returns N joint samples from the list of RVs;
+        if location is one of the locations of the posterior,
```

Comment:
There is some grammatical mistake in the sentence here, and I'm not 100% sure I understand what it tells me.

Maybe also add a `Parameters` section to describe `locations` and `size`.

Reply:
Oh good point! I wrote those before I wrote the code, this is why it sounds more like a shopping list than a description. I will update them!